src/main.rs

```diff
@@ -25,7 +25,8 @@ async fn main() -> Result<(), Box<dyn Error>> {
     const DEFAULT_HOSTNAME: &str = "localhost";
     const DEFAULT_USERNAME: &str = "Administrator";
 
-    pretty_env_logger::init();
+    // pretty_env_logger::init();
+    log4rs::init_file("log4rs.yml", Default::default()).unwrap();
```

Comment:
what happens if that file is not present? if you cargo run from the dir  it is there, but once we ship the distribution it might be not -- should we include a default config as part of the binary and optionally check for that logfile somewhere?